### PR TITLE
added repos to repo list

### PR DESCRIPTION
**Pull request by kierisi**
Added lessons and peer-review-metrics repositories to the current `our-repos` section, which should now cover all of the pyOpenSci repos.

### CODE REVIEW

**Comment by kierisi** [comment]
pre-commit.ci autofix

File: community/github/our-repositories.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ The software-review repository is where community package submissions are
 peer-reviewed. All submissions are made through GitHub Issues. [Learn more
 about our peer review process here.](https://www.pyopensci.org/software-peer-review/)
 
+Individuals and teams with access to this repository include: [the pyOpenSci Editorial Board](https://github.com/orgs/pyOpenSci/teams/editorial-board), [Bane Sullivan](https://github.com/banesullivan), [Derek Homeier](https://github.com/dhomeier), [Kelly L. Rowland](https://github.com/kellyrowland), [Filipe Fernandes](https://github.com/ocefpaf), [Simon Molinsky](https://github.com/SimonMolinsky), [Emily Jane McTavish](https://github.com/snacktavish), [Jonny Saunders](https://github.com/snacktavish), [Tetsuo Koyama](https://github.com/tkoyama010), and [Tom Russell](https://github.com/tomalrussell).
```
Comment (lwasser) [inline community/github/our-repositories.md:17]:
```suggestion
Teams with access to this repository: 
* [the pyOpenSci Editorial Board](https://github.com/orgs/pyOpenSci/teams/editorial-board)
```

File: community/github/our-repositories.md [hunk 2]
```diff
@@ -30,19 +32,25 @@ peer review triage team as they manage our open peer review process. It also
 details our peer review policies, partnerships, and the templates used in the
 review process.
 
+Individuals and teams with access to this repository include: [the pyOpenSci Editorial Board](https://github.com/orgs/pyOpenSci/teams/editorial-board) and [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers).
```
Comment (lwasser) [inline community/github/our-repositories.md:35]:
```suggestion
Individuals and teams with access to this repository include: 
* [the pyOpenSci Editorial Board](https://github.com/orgs/pyOpenSci/teams/editorial-board)
* [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers).
```

File: community/github/our-repositories.md [hunk 3]
```diff
@@ -30,19 +32,25 @@ peer review triage team as they manage our open peer review process. It also
...
+
 ## Python-package-guide repository
 
 The [python-package-guide
 repository](https://www.pyopensci.org/python-package-guide/) contains our
 community-developed guidelines and tutorials on Python packaging. These
 resources are beginner-friendly and reflect Python packaging best practices.
 
+Individuals and teams with access to this repository include: [the pyOpenSci Packaging Council](https://github.com/orgs/pyOpenSci/teams/packaging-council), [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers), [Alex Batisse](https://github.com/Batalex), [Felipe Moreno](https://github.com/flpm), [Jackson Burns](https://github.com/JacksonBurns), [Jeremy Paige](https://github.com/ucodery), and [Carol Willing](https://github.com/willingc).
```
Comment (lwasser) [inline community/github/our-repositories.md:44]:
```suggestion
Teams with access to this repository: 
* [the pyOpenSci Packaging Council](https://github.com/orgs/pyOpenSci/teams/packaging-council)
* [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers)
```

File: community/github/our-repositories.md [hunk 4]
```diff
@@ -30,19 +32,25 @@ peer review triage team as they manage our open peer review process. It also
...
+
 ## [pyosMeta repository](https://github.com/pyOpenSci/pyosMeta)
 
 The pyosMeta repository contains a Python package published on PyPI that we use
 to track our package review and contributor data. This data is used in a GitHub
 action to update our website.
 
+Individuals and teams with access to this repository include: [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers), [Steven Silvester](https://github.com/blink1073), [Mike Sarahan](https://github.com/msarahan), and [Carol Willing](https://github.com/willingc).
```
Comment (lwasser) [inline community/github/our-repositories.md:52]:
```suggestion
Teams with access to this repository: 
* [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers)
```

File: community/github/our-repositories.md [hunk 5]
```diff
@@ -58,6 +66,8 @@ automatically using a GitHub action workflow that is supported by the pyosMeta
 Python package discussed above. The workflow runs every other week but can be
 triggered manually as a **workflow dispatch**.
 
+Individuals and teams with access to this repository include: [the pyOpenSci Editorial Board](https://github.com/orgs/pyOpenSci/teams/editorial-board), [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers), [Jesse Mostipal](https://github.com/kierisi), [Mike Sarahan](https://github.com/msarahan), [pyosbot](https://github.com/pyosbot), [Thomas J. Fan](https://github.com/thomasjpfan), [Carol Willing](https://github.com/willingc), and [Ivan Ogasawara](https://github.com/xmnlab).
```
Comment (lwasser) [inline community/github/our-repositories.md:69]:
```suggestion
Teams with access to this repository: 
* [the pyOpenSci Editorial Board](https://github.com/orgs/pyOpenSci/teams/editorial-board)
* [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers)
```

File: community/github/our-repositories.md [hunk 6]
```diff
@@ -110,16 +120,34 @@ of our repositories that use the theme.
 Creating a theme was inspired by the
 [2i2c Sphinx theme](https://sphinx-2i2c-theme.readthedocs.io/en/latest/).
 
+Individuals and teams with access to this repository include: [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers) and [Revathy Venugopal](https://github.com/Revathyvenugopal162).
```
Comment (lwasser) [inline community/github/our-repositories.md:123]:
```suggestion
Teams with access to this repository: 
* [the pyOpenSci Repository Maintainers team](https://github.com/orgs/pyOpenSci/teams/pyos-repo-maintainers)
```

File: community/github/our-repositories.md [hunk 7]
```diff
@@ -110,16 +120,34 @@ of our repositories that use the theme.
...
+
 ## [Handbook repository](https://github.com/pyOpenSci/handbook)
 
 **Platform:** Sphinx book running the `pydata_sphinx_theme`
 
 This is where we store our organization governance, code of conduct, and
 processes around how we operate as an organization.
 
+Individuals and teams with access to this repository include: [Karen Cranston](https://github.com/kcranston), [Jesse Mostipak](https://github.com/kierisi), [Tracy Teal](https://github.com/tracykteal), and [Carol Willing](https://github.com/willingc).
```
Comment (lwasser) [inline community/github/our-repositories.md:132]:
```suggestion
The [pyOpenSci Executive Council](https://www.pyopensci.org/our-community/index.html#executive-council-leadership--staff) has access to this repo.  
```

File: community/github/our-repositories.md [hunk 8]
```diff
@@ -110,16 +120,34 @@ of our repositories that use the theme.
...
+
 ## [pyosPackage repository](https://github.com/pyOpenSci/pyosPackage)
 
 The pyosPackage repo contains an example pure-Python package that complements
 our package guide & tutorials. We will build this package example out over time
 for folks that just want to see a working package without creating one
 themselves.
+
+Individuals and teams with access to this repository include: [the pyOpenSci Packaging Council](https://github.com/orgs/pyOpenSci/teams/packaging-council).
```
Comment (lwasser) [inline community/github/our-repositories.md:141]:
```suggestion
Teams with access to this repository: 
* [the pyOpenSci Packaging Council](https://github.com/orgs/pyOpenSci/teams/packaging-council).
```

File: community/github/our-repositories.md [hunk 9]
```diff
@@ -110,16 +120,34 @@ of our repositories that use the theme.
...
+
+The pyOpenSci peer review metrics repository contains the code for a dashboard created using [MyST Markdown](https://mystmd.org/). Myst-md is a community developed tool that makes it easier for scientists to create fully reproducible (and interactive) workflows and reports that are easily shared.
+
+No individuals or teams have access to this repository.
+
+## [lessons repository](https://github.com/pyOpenSci/lessons)
+
+pyOpenSci is devoted to building diverse, supportive community around the Python open source tools that drive open science. The lessons repository contains the source files for all of the [pyOpenSci tutorials](https://github.com/pyOpenSci/lessons).
+
+No individuals or teams have access to this repository.
```
Comment (lwasser) [inline community/github/our-repositories.md:153]:
```suggestion
The [pyOpenSci Lesson Development Team](https://github.com/orgs/pyOpenSci/teams/lesson-development) has access to this repo. 
```

**Comment by lwasser** [comment]
pre-commit.ci autofix

**Pull request merged**